src/openeo_grass_gis_driver/collection_information.py

```diff
@@ -255,8 +223,9 @@ def get(self, name):
 
         coordinate_transform_extent_to_EPSG_4326(crs=crs, extent=extent)
 
-        properties = (CollectionProperties(eo_platform="Sentinel-2",
-                                           eo_instrument="Sentinel-2",
+        # GRASS / actinia do not yet report platform and instrument
+        properties = (CollectionProperties(eo_platform="unknown",
```

Comment:
Can we somehow look for `s2`,  `S2` and `Sentinel-2` and if contained, we change it to Sentinel-2? I know it is not supernice and s2 might appear in another name as well, but I think for now it is the most common use case and would look better than `unknown`

Reply:
Yes, should be possible, I will give it a try.

Reply:
The platforms landsat-5, landsat-7, landsat-8, sentinel-1, sentinel-2 with their instruments are now recognized.

Reply:
nice!